apps/builder/app/shared/html.test.tsx

```diff
@@ -222,3 +222,25 @@ test("generate style attribute as local styles", () => {
     )
   );
 });
+
+test("paste svg as html embed", () => {
+  expect(
+    generateFragmentFromHtml(`
+      <div>
+        <svg viewBox="0 0 20 20">
+          <rect x="5" y="5" width="10" height="10" />
+        </svg>
+      </div>
+    `)
+  ).toEqual(
+    renderTemplate(
+      <ws.element ws:tag="div">
+        <$.HtmlEmbed
+          code={`<svg viewBox="0 0 20 20">
```

Comment:
would be great to use some stripIndent utility or something to be able to nicely format this stuff in the tests

Reply:
I test here specifically how result is formatted and it shouldn't be prettified.